pkgs/development/python-modules/mapbox-earcut/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3Packages
+, fetchFromGitHub
+, setuptools
+}:
+
+with python3Packages; buildPythonPackage rec {
+  pname = "mapbox-earcut";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "skogler";
+    repo = "mapbox_earcut_python";
+    rev = "257c2c82819acf72b2b757f200c160e55b9427fc";
+    sha256 = "14z2vj13a1qcaqc6jvg00f367la5470bclvfi9mmlvpac3hhhywd";
+    };
+
+    nativeBuildInputs = [ setuptools pybind11 ];
+    propagatedBuildInputs = [ numpy ];
+
+    # Some test failed, some passed...
+    checkInputs = [ pytestCheckHook ];
+    doCheck = false;
```

Comment:
Can we ignore the failing ones instead?

Reply:
yes -> c04fc24542ff1bac9c1b909df24ea98e3a3fbce7